app/config.py

```diff
@@ -35,6 +35,34 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
     return literal_eval(value)
 
 
+def get_env_dict(env_var: str, default_factory: Callable = None) -> dict[str, str]:
```

Comment:
Worth leaving default as empty dict directly?

Reply:
Or even better, not allowing to pass it as it will always be a dict

Reply:
👍